### PR TITLE
EASY-1469: More form fields

### PR DESCRIPTION
**Pull request by rvanheest**
Part of EASY-1469

#### When applied it will
* implement more form fields
* add hot module reloading for CSS

@DANS-KNAW/easy for review

just a couple of examples:

image

image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 node_modules
 .idea
 .vagrant
+dist
```
Comment (janvanmansum) [inline .gitignore]: What is the `dist` folder for?

Comment (rvanheest) [reply .gitignore]: Mmm, I keep getting the `dist` folder. Sometimes when I'm compiling with an error, sometimes when not having any compile errors. However, before this PR I did not have anything like this. I'll verify that and take a close look as to what has changed.

Comment (rvanheest) [reply .gitignore]: I have reduced this problem to Webstorm (currently 2018.1.2). I'll investigate further.

Comment (rvanheest) [reply .gitignore]: https://youtrack.jetbrains.com/issue/WEB-32527

for now, let's keep this line in `.gitignore`

**Pull request merged**